source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -226,19 +226,28 @@
                     name: groupNames[key],
                     items: _.map(items, function (item) {
                         itemNumber++;
+                        var rowSelected = ko.observable(false);
                         var valuesByItem = {
                             key: item.Key,
                             text: itemNumber + " - " + item.Text,
                             description: item.Description,
+                            isRowSelected: rowSelected,
                             values: _.map(valuesByKeyCollection, function (valuesByKey) {
                                 var valueItem = {
                                     calificationId: valuesByKey.calificationColumn.calificationId,
                                     value: ko.observable(valuesByKey[item.Key] || ""),
                                     editable: self.isValueEditable(valuesByKey),
+                                    isSelected: rowSelected,
                                     owner: valuesByKey.calificationColumn.owner,
                                     showValue: _.find(self.califications, function (calification) {
                                         return calification.id == valuesByKey.calificationColumn.calificationId;
                                     }).show,
+                                    onBlur: function (data, event) {
+                                        data.isSelected(false);
+                                    },
+                                    onFocus: function (data, event) {
+                                        data.isSelected(true);
+                                    }
```

Comment:
I'd really really suggest breaking this huge map into different parts. It's wayyyy to big.

```javascript
 var groups = _.chain(...).groupBy();
 return _.map(groups, getBindableItems)

function getItemsFromGroups(items, key) {
  // ...
  return _.map(items, getValues);
}

function getValues(valuesByKeyCollection) {
  var valueItem = {
  // ...
  };
  // ...
  return valueItem;
}
```

I may have made mistakes above but you get the idea.